### PR TITLE
Fix wrong SVG positions due to transform attribute

### LINKED ISSUE #1455: SVG Import does not correctly position elements not contained with a separate layer
In LaserGRBL v4.3.0, when importing an SVG file containing elements that have been placed in the main body of the SVG document rather than into a separate layer, it seems to not correctly position those elements.

Simple reproduction attached in zip file (archive only contains a single SVG file, but Github won't let you upload a file with SVG extension - doh!), in which the grid lines are placed in a separate layer ("Layer 1"), with translate(0.25,60.25) attribute. Text and outline are not contained within that layer. The SVG renders correctly in Inkscape and when loaded in a browser such as Microsoft Edge, but incorrectly in LaserGRBL as shown in screenshots.

[LaserGRBL-LayerBug.zip](https://github.com/arkypita/LaserGRBL/files/6451815/LaserGRBL-LayerBug.zip)

Bug_LaserGRBL
Bug_Edge
Bug_Inkscape

### PR DESCRIPTION
**Pull request by falias4**
TL;DR: I simply replaced the functions regarding the transform-attribute with the latest ones from GRBL-Plotter.

---
As discussed in #1455 the `GCodeFromSVG.cs` is quite old and a whole rework would take much time.
So I specifically searched for the reported issue and simply replaced the functions with the new ones.

Basically only had to change the `ParseTransform` so that it returns the `Matrix` instead of a `bool`.
Other small changes are due to changes of function names (casing) and that they are static now.

Should fix #1455 and #1651

### CODE REVIEW

**Comment by arkypita** [comment]
Included in v4.6.3 (pre-release) https://github.com/arkypita/LaserGRBL/releases/download/v4.6.3/install.exe
